lib/Twig/Loader/Filesystem.php

```diff
@@ -168,18 +167,33 @@ public function isFresh($name, $time)
 
     protected function findTemplate($name)
     {
+        $throw = func_num_args() > 1 ? func_get_arg(1) : true;
```

Comment:
We cannot add an additional argument without breaking BC here. The argument is added in 2.0 though.